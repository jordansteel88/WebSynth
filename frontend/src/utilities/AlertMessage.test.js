import React from "react";
import { render } from "@testing-library/react";
import AlertMessage from "./AlertMessage";

it("renders without crashing", function() {
  render(<AlertMessage />);
});

it("matches snapshot for danger", function() {
  let messages = ["Everything is broken", "Run for the hills"];
  const { asFragment } = render(<AlertMessage type="danger" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot for success", function() {
  let messages = ["Everything is awesome!"];
  const { asFragment } = render(<AlertMessage type="success" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});