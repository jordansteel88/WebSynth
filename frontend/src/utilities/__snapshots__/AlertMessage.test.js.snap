// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for danger 1`] = `
<DocumentFragment>
  <div>
    <div
      class="pb-0 small alert alert-danger fade"
      role="alert"
    >
      <p>
        Everything is broken
      </p>
      <p>
        Run for the hills
      </p>
    </div>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot for success 1`] = `
<DocumentFragment>
  <div>
    <div
      class="pb-0 small alert alert-success fade"
      role="alert"
    >
      <p>
        Everything is awesome!
      </p>
    </div>
  </div>
</DocumentFragment>
`;
